lib/cn/csched_sp3.c

```diff
@@ -828,12 +828,13 @@ sp3_dirty_node_locked(struct sp3 *sp, struct cn_tree_node *tn)
      */
 
     if (cn_node_isroot(tn)) {
+        uint jobs_max = sp->samp_reduce ? 1 : 3;
```

Comment:
this is a great idea.

Reply:
Thanks!  Best I can do on short notice.  It won't be as effective if you have a single-node doing zspills or using PMEM for staging, but I have a ticket open for dealing with the latter, and using wlen instead of clen for splitting addresses the former.  At least for now...